### ISSUE TITLE
Hard to edit when importing a track from relation

### ISSUE THREAD

**Issue by valleyofdawn** [IsraelHikingMap/Site#479]
Moving a single node collapses the entire track instead of affecting only the neighboring sections.
e.g.  סינגל קצרין

**Comment by HarelM** [comment]
Can you elaborate the exact scenario?
What you did, what happened and what you expected to happen?

**Comment by valleyofdawn** [comment]
Searched for  "סינגל קצרין"
Converted it to a track using the "+" button.
Tried to edit this track by dragging a node along it
Expected the track to remain largely the same except for accommodating the new node.
Instead the track collapsed to include only the start and end point and the dragged node with the shortest way connecting the 3 nodes. Went from 14 Km to 3.

**Comment by HarelM** [comment]
As far as I can tell this was the behavior when using search previously, which means this is not a bug but a feature request.
I understand the requirement but I believe we have bigger fish to fry.
Hopefully after I'll finish with the POIs with no names.

**Comment by HarelM** [comment]
A solution to the problem that I can think of is to do the splitting algorithm much like we do for files when loading them.
The problem I see with this solution is that the route data is sent twice - once for geojson and the other for an internal data structure - this data can be big sometimes and might take a few seconds to be sent.
I prefer not to do the splitting on the client side since It will create a different results between the server and the client.
Bottom line, I'll duplicate the information and we'll see if we need to improve the performance later on...
